src/ResourceManagement/Automation/AutomationManagement/Generated/Models/DscNode.cs

```diff
@@ -41,6 +43,17 @@ public Guid AccountId
             set { this._accountId = value; }
         }
         
+        private IList<DscNodeExtensionHandlerAssociationProperty> _extensionHandler;
```

Comment:
You only require change in 3 files to go. Right? DscNodeOperations.cs, DscNode.cs, DscNodeExtensionHandlerAssociationProperty.cs. Right?

Reply:
yes just these 3